pages/common/mitmproxy.md

```diff
@@ -0,0 +1,11 @@
+# mitmproxy
+
+> An interactive man-in-the-middle HTTP proxy.
```

Comment:
That was a good explanation. I wasn't all too sure but you confirmed that mitm really only is used in context of a mitm attack. Viewed in that light, it really makes sense to have it spelled out like you have it here.